.github/workflows/kind.yml

```diff
@@ -108,6 +108,11 @@ jobs:
           make kind-bootstrap-cluster-dev
           KUBECONFIG=${PWD}/kubeconfig_managed make kind-ensure-sa
 
+      - name: OCM-policy namespace Tests
```

Comment:
Should I put this on a separate line? It takes 20 minutes.

Reply:
Yes--it should run in its own set of steps like hosted mode (and, in my opinion, only run on `latest`).

Reply:
Great I thought that too only `latest`